macrosynergy/management/dq.py

```diff
@@ -261,6 +266,33 @@ def _fetch(self, endpoint: str = "/groups", select: str = "groups",
 
         return results
 
+    def _fetch_threading(self, endpoint, params: dict):
```

Comment:
Replace the `_fetch` method with the `_fetch_threading`?

Reply:
Kept so I can later replicate the logging complete in the _fetch method. Once that is complete, the method will be removed.